lansweeper.json

```diff
@@ -0,0 +1,689 @@
+{
+    "appid": "2bde184d-aace-476b-b584-1dab0926d25d",
+    "name": "Lansweeper",
+    "description": "This app integrates with Lansweeper to perform investigative actions",
+    "type": "investigative",
+    "product_vendor": "Lansweeper",
+    "logo": "logo_lansweeper.svg",
+    "logo_dark": "logo_lansweeper_dark.svg",
+    "product_name": "Lansweeper",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Lansweeper",
+    "license": "Copyright (c) Lansweeper, 2022",
```

Comment:
if we've made code changes (which it looks like we have) we should also include Splunk as a license author. see for example: https://github.com/splunk-soar-connectors/code42v2/pull/1/files?w=1#diff-f91890de7d72e73187b478bbb8a5000574522fd4ae1b6acb25b7b2e9a97ac44fR14

Reply:
Yes @rgil-splunk, we kept both licenses in Code42v2 because it is a Splunk-supported app. However, we are now in discussions with Junlin on the Developer-supported app’s licensing that should we keep Splunk license or not along with the third-party license. For now, we are keeping only third-party licenses as per Junlin’s suggestion.

We will update you if there are any updates.

CC: @jpuvvula

Reply:
make sure to loop in the legal team in that discussion if you haven't already. the guidance to include multiple copyright holders came from them